code/ai/aigoals.cpp

```diff
@@ -150,25 +156,26 @@ void ai_maybe_add_form_goal(wing* wingp)
 
 		aip = &Ai_info[Ships[wingp->ship_index[j]].ai_index];
 		// don't process Player_ship
-		if (aip == Player_ai || (MULTIPLAYER_MASTER && Objects[Ships[aip->shipnum].objnum].flags[Object::Object_Flags::Player_ship])) {
+		if (aip == Player_ai || ((Game_mode & GM_MULTIPLAYER) && Objects[Ships[aip->shipnum].objnum].flags[Object::Object_Flags::Player_ship])) {
```

Comment:
I don't even think you need the multiplayer check here now.  In single-player, only the player ship is the player ship.

Reply:
So just `if (Objects[Ships[aip->shipnum].objnum].flags[Object::Object_Flags::Player_ship])` to check if it is a player ship?

Reply:
Right.